File: SportAppServer/SportAppServer/SportAppServer/Support2026/Application/Dto/NewsDto.cs
using Newtonsoft.Json;
using System.Text.Json.Serialization;

namespace SportAppServer.Support2026.Application.Dto
{
    public class NewsDto
    {
        [JsonProperty("date_time")]
        public DateTime DateTime { get; set; }

        [JsonProperty("sport")]
        public string Sport { get; set; }

        [JsonProperty("title")]
        public string Title { get; set; }

        [JsonProperty("image_id")]
        public string ImageId { get; set; }

        [JsonProperty("text")]
        public string ArticleText { get; set; }      
    }
}

File: app/src/main/java/com/example/sportapp/support2026/features/news/data/dto/components/NewsDto.kt
package com.example.sportapp.support2026.features.news.news.data.dto.components

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
class NewsDto (
    @SerialName("dateTime") val dateTime: String?,
    @SerialName("sport") val sport: String?,
    @SerialName("imageId") val imageId: String?,
    @SerialName("articleText") val text: String?,
    @SerialName("title") val title: String?,

)


